.github/workflows/continous-integration.yml

```diff
@@ -293,9 +293,9 @@ jobs:
         if: needs.changes.outputs.backend == 'true' && matrix.os == 'ubuntu-22.04'
         run: |
           sudo apt-get -y install libpq-dev
-          make install-full | tee .output
           if grep 'The lock file is not up to date' .output; then exit 1; fi
```

Comment:
This needs to happen after make install-full.

Reply:
Oh right, making this change